codec-http2/src/test/java/io/netty/handler/codec/http2/Http2FrameCodecTest.java

```diff
@@ -150,22 +149,26 @@ public void entityRequestEntityResponse() throws Exception {
         // Release hello to emulate ByteToMessageDecoder
         hello.release();
         Http2DataFrame inboundData = inboundHandler.readInbound();
-        assertEquals(releaseLater(new DefaultHttp2DataFrame(bb("hello"), true, 31).streamId(stream.id())),
-                     releaseLater(inboundData));
+        Http2DataFrame expected = new DefaultHttp2DataFrame(bb("hello"), true, 31).streamId(stream.id());
+        assertEquals(expected, inboundData);
         assertEquals(1, inboundData.refCnt());
+        expected.release();
         assertNull(inboundHandler.readInbound());
 
         inboundHandler.writeOutbound(new DefaultHttp2HeadersFrame(response, false).streamId(stream.id()));
         verify(frameWriter).writeHeaders(eq(http2HandlerCtx), eq(1), eq(response), anyInt(),
```

Comment:
No don't worry about for this PR. Just noticed it bcz I was attempting to step through and look at reference counts. https://github.com/netty/netty/issues/6036